src/ovh-angular-sidebar-menu.provider.js

```diff
@@ -379,6 +379,12 @@ angular.module("ovh-angular-sidebar-menu").provider("SidebarMenu", function () {
                     }
                     if (stateInfos.included) {
                         return item.loadSubItems().then(function () {
+                            // Automatically close same level opened item if it is not current one
+                            var openedItem = _.find(sidebarMenuService.getAllMenuItems(), { isOpen: true, level: item.level });
```

Comment:
No ES6 ?

Reply:
According to the `Gruntfile.js` configuration file, you can't use ES6 syntax (missing a transpile task).

Reply:
Our targets do support `const` though but it's a dangerous path so I get we stick to es5 😄 

http://kangax.github.io/compat-table/es6/